circle.yml

```diff
@@ -13,10 +18,12 @@ test:
     - set -o pipefail &&
       swiftlint lint --strict --reporter json |
       tee $CIRCLE_ARTIFACTS/swiftlint-report.json
-    - bin/test iOS
-    - bin/test UIKit-iOS
-    - bin/test tvOS
-    - bin/test UIKit-tvOS
+    - bin/test iOS 9.3
+    - bin/test iOS 10.1
+    - bin/test UIKit-iOS 9.3
+    - bin/test UIKit-iOS 10.1
+    - bin/test tvOS 10.1
+    - bin/test UIKit-tvOS 10.1
```

Comment:
here's where we are running tests for multiple versions of ios/tvos. 

any better ideas?